Woof/Woof.xcodeproj/project.pbxproj

```diff
@@ -862,7 +862,9 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = FDE0CEB32A00F8390023F784 /* Build configuration list for PBXNativeTarget "Woof" */;
 			buildPhases = (
+				198A7E0B2AA86C6400903FED /* Insert salt and API key */,
 				FDE0CEA12A00F8370023F784 /* Sources */,
+				198A7E0C2AA8819800903FED /* Erase salt and API key */,
 				FDE0CEA22A00F8370023F784 /* Frameworks */,
 				FDE0CEA32A00F8370023F784 /* Resources */,
 				FDE0CEB82A011BA10023F784 /* Swiftlint */,
```

Comment:
Could we move the new scripts down, after the default settings?
For me, it will be easier to find and maintain them in the future.

```suggestion
				FDE0CEA12A00F8370023F784 /* Sources */,
				FDE0CEA22A00F8370023F784 /* Frameworks */,
				FDE0CEA32A00F8370023F784 /* Resources */,
				198A7E0C2AA8819800903FED /* Erase salt and API key */,
				198A7E0B2AA86C6400903FED /* Insert salt and API key */,
				FDE0CEB82A011BA10023F784 /* Swiftlint */,
```

Reply:
It matters to insert `salt` and `obfuscatedKey` into the source file before `Compile sources` phase, otherwise the compiled program doesn't contain these values. The initial state of the source file is restored right away after `Compile sources` phase. I think it's reasonable.

Reply:
I got the idea, as we discussed on call we keep the original version.